org/postgresql/jdbc4/AbstractJdbc4Connection.java

```diff
@@ -99,7 +99,7 @@ public Struct createStruct(String typeName, Object[] attributes) throws SQLExcep
     public Array createArrayOf(String typeName, Object[] elements) throws SQLException
     {
         checkClosed();
-        int oid = getTypeInfo().getPGArrayType(typeName);
+        int oid = getTypeInfo().getPGArrayType(typeName.toLowerCase());
```

Comment:
You should use different branches for different PRs.
It is not recommended to create PRs out of a master branch.
In other words, you create a branch for "default encoding" and a branch for "name case".

In present situation it would be easier to just `git reset --hard 8dae0ae5df6ffaa484d0d6ff69aaa33f5d1d713b && git push -f origin master`. This will update PR with to have just a single commit, then you could merge it in.